<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hello, World!</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Serial port : </string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="serialPortComboBox"/>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Go!</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="fusComboBox">
        <item>
         <property name="text">
          <string>RightRear</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>RightCentre</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>RightFront</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>LeftRear</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>LeftCentre</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>LeftFront</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Fus:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="motionComboBox">
        <item>
         <property name="text">
          <string>Romb</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>RotateLeft</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>RotateRight</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Forward</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Calibre</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Step</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Motion</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>561</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
